2-members.md

```diff
@@ -69,8 +69,34 @@ banner_color: style2
    </section>
 </section>
 
-<!-- 2 -->
-<section id="two">
+
+<section id="two" class="spotlights">
+    <section>
+     <a href="generic.html" class="image">
+      <img src="assets/images/members/HamtaBardool.jpg" alt="" data-position="center center" />
+     </a>
+     <div class="content">
+      <div class="inner">
+       <header class="major">
+        <h3>Hamta Bardool</h3>
+       </header>
+       <li>Joined in Summer 2024.</li>
+       <li>Ph.D., Chemical Engineering, Purdue University/Shiraz University, 2023.</li>
+                   <p><b>Research topic</b>: Optimization,Process System Engineering, and Catalyst.</p>
```

Comment:
I think we are not working on `catalyst `but on `catalysis`. Also, a space is missing after the comma between Optimization and Process. I would say out topic is: 
`Reactor and catalysis modeling, simulation, and optimization.`

Reply:
Sure, Prof. Bernal, 
I fixed it.